### ISSUE TITLE
Serial Protocal Initialize block is broken

### ISSUE THREAD

**Issue by MatzElectronics** [parallaxinc/solo#195]
The serial initialize block under the Protocols menu is broken.  It freezes the workspace.

Here's a stack trace:
```
Uncaught Error: Field "ck_bit0" not found.
    at Blockly.BlockSvg.Blockly.Block.setFieldValue (blockly_compressed.js:1477)
    at Blockly.InsertionMarkerManager.createMarkerBlock_ (blockly_compressed.js:1133)
    at new Blockly.InsertionMarkerManager (blockly_compressed.js:1128)
    at new Blockly.BlockDragger (blockly_compressed.js:1148)
    at Blockly.TouchGesture.Blockly.Gesture.startDraggingBlock_ (blockly_compressed.js:1199)
    at Blockly.TouchGesture.Blockly.Gesture.updateIsDraggingBlock_ (blockly_compressed.js:1196)
    at Blockly.TouchGesture.Blockly.Gesture.updateIsDragging_ (blockly_compressed.js:1198)
    at Blockly.TouchGesture.Blockly.Gesture.updateFromEvent_ (blockly_compressed.js:1193)
    at Blockly.TouchGesture.Blockly.Gesture.handleMove (blockly_compressed.js:1204)
    at Blockly.TouchGesture.handleMove
```

**Comment by zfi** [comment]
This issue cannot be tested until issue #231 is corrected.

**Comment by MatzElectronics** [comment]
I see the typo that's causing this, so it should be a simple fix - I'll
correct it this afternoon when I'm free.

**Comment by MatzElectronics** [comment]
This is effectively the same issue as #224

**Comment by zfi** [comment]
This is working as expected now.
Closing the issue.

**Comment by MatzElectronics** [comment]
Reopening - discovered with @kgracey that the serial init block is not properly saving it's pins - they reload in the blocks okay, but not for code generation.  If you re-open a previously saved project, the compile will fail until you re-select the pins in the init block.

**Comment by zfi** [comment]
Closing the issue again. The pin persistence issue is a new issue and should be addressed in #246.
